sdcm/tester.py

```diff
@@ -374,6 +374,19 @@ def __init__(self, *args, **kwargs):  # pylint: disable=too-many-statements,too-
         time.sleep(0.5)
         InfoEvent(message=f"TEST_START test_id={self.test_config.test_id()}").publish()
 
+    def _init_test_duration(self):
+        self._stress_duration: int = self.params.get('stress_duration')
+        if self._stress_duration:
```

Comment:
Same here. Why the `prepare_duration` depends on the `stress_duration`?
Why I cannot use one without another? And why the `prepare` one gets ignored silently if second new option is not set?

Reply:
answered above

Reply:
This is still actual having the same change be done in the `sct_config.py`.

Reply:
having it based `stress_duration` is o.k., since we want the default behavior to be left alone, and there no point of using `prepare_duration` on it's own